brave-spark-framework-filter/README.md

```diff
@@ -0,0 +1,56 @@
+# brave-spark-framework-filter
+
+The module contains a Spark framework filter implementing to deal with server
+side integration: Getting existing span/trace state from request,
+create and submit span with `sr`', `ss` annotations.
+
+
+Here is a use Example:
+
+First,add dependency
+
+```
+<dependency>
+    <groupId>io.zipkin.brave</groupId>
+    <artifactId>brave-spark-framework-filter</artifactId>
+    <version>4.0.6</version>
+</dependency>
+        
+```
+Second,configure Brave
+```
+@Configuration
+public class BraveConfiguration {
+
+    /*@Bean 
+    Sender sender() {
+        return OkHttpSender.create("http://127.0.0.1:9411/api/v1/spans");
+    }*/
+
+    @Bean
+    public Reporter reporter() {
+        return new MyLogReporter();// I just report to log files,then use flume to gather.
```

Comment:
:) nice to know, but I don't think we want to steer people towards custom reporters when introducing spark.  how about looking at the other READMEs.. I think most of them focus on what's particular to the framework itself.